yogstation/code/game/machinery/computer/arcade.dm

```diff
@@ -0,0 +1,51 @@
+/obj/machinery/computer/arcade
+	prizes = list(
```

Comment:
Would it not be much better to just do

`/obj/machinery/computer/arcade/Initialize()
	.=..()
	prizes[/obj/item/grenade/plastic/glitterbomb/pink] = 1`